mlcolvar/cvs/cv.py

```diff
@@ -192,10 +196,18 @@ def configure_optimizers(self):
         torch.optim
             Torch optimizer
         """
+
         optimizer = getattr(torch.optim, self._optimizer_name)(
             self.parameters(), **self.optimizer_kwargs
         )
-        return optimizer
+
+        if self.lr_scheduler_kwargs:
+            if self._lr_scheduler_name is None:
+                self._lr_scheduler_name = self.lr_scheduler_kwargs.pop('scheduler')
+            lr_scheduler = self._lr_scheduler_name(optimizer, **self.lr_scheduler_kwargs)
```

Comment:
If we remove ``self._lr_scheduler_name``, this would become something like
```python
scheduler_cls = self.lr_scheduler_kwargs['scheduler']
scheduler_kwargs = {k: v for k, v in self.lr_scheduler_kwargs.items() if k != 'scheduler'}
lr_scheduler = scheduler_cls(**scheduler_kwargs)
```